plugins/catalog-react/src/components/EntityTable/EntityTable.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { Table, TableColumn } from '@backstage/core';
+import { makeStyles } from '@material-ui/core';
+import React from 'react';
+import * as columnFactories from './columns';
+import { componentEntityColumns, systemEntityColumns } from './presets';
+
+type Props<T extends Entity> = {
+  title: string;
+  variant?: string;
```

Comment:
```suggestion
  variant?: 'gridItem';
```

Reply:
I like to use an enum type here. We should fix that for all components that use variant, but I will just do it in the components of this PR for now to keep the changes small. Should also be fixed in the InfoCard itself.

Reply:
Created #4558